### PR TITLE
Add EL8 instructions

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: _includes/manuals/nightly/2.1_quickstart_installation.md [hunk 1]
```diff
@@ -58,6 +60,17 @@ sudo yum -y install https://yum.puppet.com/puppet6-release-el-7.noarch.rpm
 {% endhighlight %}
 </div>
 
+<div class="quickstart_os quickstart_os_rhel8 quickstart_os_el8">
```
Comment (evgeni) [inline _includes/manuals/nightly/2.1_quickstart_installation.md:63]: Should this step also point to the now required "configmanagement-ansible" repo? Or is that optional due to the fact only a plugin needs ansible?

Comment (ehelms) [reply _includes/manuals/nightly/2.1_quickstart_installation.md:63]: The latter I should think.

Comment (evgeni) [reply _includes/manuals/nightly/2.1_quickstart_installation.md:63]: wfm.

**Review by evgeni — APPROVED** [review]
will you do the cherry pick to 2.1 in a separate PR?

**Comment by tbrisker** [comment]
@ehelms please apply the same change to the 2.1 manual :)

**Comment by ehelms** [comment]
Add 2.1 manual changes

**Comment by tbrisker** [comment]
Thanks @ehelms !

**Pull request merged**